### ISSUE TITLE
Incorrect tooltip when using Spirit Breaker's Charge

### ISSUE THREAD

**Issue by tWOVgu** [jeffhill/Dota2#549]
### Hero and spell

Spirit Breaker

### Description

When using the Charge, the hero gets a Buff with the image of the hero on which the ability was used. When you click on this Buff, a message is sent to the allies' chat on which character the ability was used. If the target dies and Spirit Breaker will change the target to a neutral creep, then the tooltip will be incorrect - the technical name of the neutral creep will be displayed.
Let me know if screenshots are needed.
Thanks for your work!

**Comment by Daar543** [comment]
What shows when you charge a neutral creep by default?

**Comment by tWOVgu** [comment]
> What shows when you charge a neutral creep by default?

Yes, if I used Charge on a neutral creep, I also saw the tooltip "Run to > npc.dota.creep.neutral".
![npc dota creep neutral](https://user-images.githubusercontent.com/106872816/172036057-7e81fa47-c032-477d-b54f-695d70f5667a.png)
I think that if you fix this hint, it will solve the problem that I wrote about earlier.
For comparison, if you use the ability Charge on a creep from the lane, then the hint is correct - "Run to > Creep from the lane"
![Creep from the lane](https://user-images.githubusercontent.com/106872816/172036077-10d25b4b-d5c6-4993-8fea-b4f99d7e717d.png)
